client/src/constants/validation/schema/schema-change-ros-site.js

```diff
@@ -8,6 +8,6 @@ export const ChangeRosSiteSchema = yup
     startDate: yup.date().required('Start Date at a New Site is required'),
     positionType: yup.string().required('Position Type is required').oneOf(rosPositionTypeValues),
     employmentType: yup.string().optional().oneOf(rosEmploymentTypeValues),
-    site: yup.number('Invalid type for New Site').required('New Site is required'),
+    site: yup.number('Invalid type for New Site').required('New Site Name is required'),
```

Comment:
It seems weird that this validation is `number` but we call it `Name` or is this just a quirk of it being a dropdown?

Reply:
Yes, so the value we're keeping and validating is `siteId` = _number_; but when selecting a site from dropdown, they only see site names.
So, we're only validating `value` and not `label`